compiler/src/main/kotlin/dev/zacsweers/metro/compiler/fir/generators/ContributedInterfaceSupertypeGenerator.kt

```diff
@@ -118,6 +119,20 @@ internal class ContributedInterfaceSupertypeGenerator(session: FirSession) :
       }
     }
 
+  private fun FirRegularClassSymbol.contributesAnnotations(): Sequence<FirAnnotation> {
+    return annotations.annotationsIn(session, session.classIds.allContributesAnnotations).ifEmpty {
+      // When there are multiple, they end up getting bundled into a 'Container' type that we need
+      // to unwrap. This behavior appears to only occur for contributions from other
+      // modules/compilations.
```

Comment:
Should we gait this check to just if this class symbol is external?

Reply:
That sounds fair to me! Didn't realize that property was available 👀

Reply:
Hmm actually it looks like `isExternal` isn't compatible with this use-case -- the prop is false for the new test case when the downstream module contributions get evaluated

Reply:
Ah no you don't want that. That refers to the `external` keyword (opposite of `internal` but much rarer use. Analogous to `native` in Java). For here I think you can just check the key of the symbol and see if it's an external declaration/stub?